src/main/ruby/core/rbconfig.rb

```diff
@@ -79,6 +79,7 @@ module RbConfig
       'LIBRUBYARG'        => '',
       'NULLCMD'           => ':',
       'optflags'          => '',
+      'PATH_SEPARATOR'    => ':',
```

Comment:
Should this be `File::PATH_SEPARATOR`? I honestly don't know.

Reply:
I think it should, although on Windows there seem to be cases where they are different.
This change is also part of https://github.com/graalvm/truffleruby/pull/487/commits/817e78bffa8633ae6117c1df3c2b33da730f03d9#diff-f80ef3d6d24890a07fed9f25d3a264d4R91 but we can merge this first since it's smaller.

Reply:
I've update this to use `File::PATH_SEPARATOR`